forms-flow-theme/scss/_card.scss

```diff
@@ -0,0 +1,186 @@
+    margin: var(--spacer-150) 0;
+}
+
+.editor-header-text {
+    color: var(--ff-white);
+    text-align: center;
+    font-size: var(--font-size-sm);
+    font-weight: var(--font-weight-xl);
+    line-height: 120%
+}
+
+.white-text {
+    color: var(--ff-white) !important
+}
+
+.form-flow-wraper-right {
+    color: var(--ff-base-600);
+    font-size: var(--font-size-lg);
+    font-weight: var(--font-weight-xl);
+    padding-bottom: var(--spacer-200) !important;
+    padding-left: var(--spacer-200) !important;
+    padding-right: 36px !important;
+    background-color: var(--ff-base-200);
+    width: 97px;
+    min-height: 581px;
+    cursor: pointer;
+    height: auto;
+    position: absolute;
+    left: 94%; 
+    overflow: hidden;
+    writing-mode: vertical-rl;
+    text-align: end;
+    transform: rotate(180deg);
+    border-top-right-radius: var(--radius-0100);
+    border-bottom-right-radius: var(--radius-0100);
+    opacity: 1;
+    transition: opacity 0.5s ease;
+}
+
+.form-flow-wraper-right.visible {
+    opacity: 1;
+    left: 94%; 
+    transition: width 0.5s ease, opacity 0.5s ease;
+}
+.form-flow-wraper-right.visible {
+    animation: shrink-right 1.5s forwards;
+}
+
+@keyframes shrink-right {
+    0% {
+        opacity: 1;
+        left: 0%;
+    }
+    100% {
+        opacity: 1;
+        left: 94%;  
+    }
+}
+
+
+
+.form-flow-wraper-left {
+    margin-right: -12px;
+    color: var(--ff-base-600);
+    width: 100px;
+    font-size: var(--font-size-lg);
+    overflow: hidden;
+    font-weight: 700;
+    padding-bottom: var(--spacer-200) !important;
+    padding-left: var(--spacer-200) !important;
+    padding-right: 36px !important;
+    background-color: var(--ff-base-200);
+    min-height: 581px;
+    height: auto;
+    position: absolute;
+    right: -10%;
+    writing-mode: vertical-rl;
+    cursor: pointer;
+    text-align: end;
+    transform: rotate(180deg);
+    border-top-left-radius: var(--radius-0100);
+    border-bottom-left-radius: var(--radius-0100);
+    opacity: 0;
+    transition: opacity 0.5s ease;
+}
+
+.form-flow-wraper-left.visible {
+    opacity: 1;
+    right: -10%;
+    transition: width 0.5s ease, opacity 0.5s ease;
+}
+
+.form-flow-wraper-left.visible {
+    animation: shrink-width 1.5s forwards;
+}
+
+@keyframes shrink-width {
+    0% {
+        opacity: 1;
+        right: -10%;
+    }
+
+    100% {
+        opacity: 1;
+        right: 96%;
+    }
+}
+
+.main-edit-wrap {
+    overflow-y: hidden !important;
+}
+
+.flow-wraper {
+    position: absolute;
+    top: 94px;
+    right: -100%;
+    width: 94%;
+    opacity: 0;
+    transition: right 1.5s ease, opacity 1.5s ease; 
+}
+
+.flow-wraper.visible {
+    opacity: 1 !important;
+    right: 0;
+}
+
+
+.form-wraper {
+    position: relative;
+    right: 100%; 
+    width: 93%;
+    opacity: 0;
+    transition: right 1.5s ease, opacity 1.5s ease; 
+}
+
+.form-wraper.visible {
+    opacity: 1 !important;
+    right: 0;
+}
```

Comment:
Bring nesting of classes here.

Delete duplicate classes @Josephalexantony-aot